harness/determined/exec/prep_container.py

```diff
@@ -255,6 +267,12 @@ def do_proxy(sess: Session, allocation_id: str) -> None:
 
     if args.resources:
         det.ResourcesInfo._by_inspection()._to_file()
+        for process in gpu.get_gpu_processes():
+            logging.warning(
+                f"process {process.process_name} "
+                f"with pid {process.pid} "
+                f"is using gpu with uuid {process.gpu_uuid}"
```

Comment:
perhaps expand a bit more on why the user should care about this

Reply:
I have no idea why a user might care about this. Did the person that requested this feature explain why? If not - or if that's not the only possible reason - all I can do is present the information.
There's more information that I can pull out of `nvidia-smi`, if that's worthwhile.